views/components/BarGraphTooltip/helpers.js

```diff
@@ -1,8 +1,13 @@
-import { N_A, TOTAL_LABEL } from '../../../constants'
+import { N_A, TOTAL_LABEL, TOTALS } from '../../../constants'
 
-const formatTooltipData = (studyCounts = 0, studyTargets) =>
+export const formatTooltipData = (studyCounts = 0, studyTargets) =>
   !!studyTargets ? `${studyCounts} / ${studyTargets}` : `${studyCounts}`
 
+export const formatTotalsTooltipData = (
+  { siteCount, siteTarget },
+  displayTargets
```

Comment:
is this meant to be a boolean that allows us to determine whether or not to display targets? If so, might be worth naming it `shouldDisplayTarget` to suggest that and passing in a boolean so you don't need the `!!` on the next line.

Reply:
Thanks for the tip.
https://github.com/AMP-SCZ/dpdash/pull/266/commits/bd75abb4ec985aa9d3675d9c9ce18f4c270d3647